src/main/resources/db/migration/V1_38__arena_forhaandsorientering.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE ARENA_FORHAANDSORIENTERING (
+    ARENAAKTIVITET_ID NVARCHAR2(255) NOT NULL,
```

Comment:
Hvorfor ikke putte alle forhåndsorienteringer i en tabell isteden for at de som hører til arenaaktiviteter er her og de andre er i aktivitettabellen?

Reply:
De andre kan ha forgen key/ tett kobling mot aktivitet. 
Tenker også at vi kanskje burde flytte ut arena i egen app

Reply:
Enig med Håkon her. Føler også at endepunktene blir såpass ulike at det gir mest mening å ha det i en egen tabell